src/main/java/uk/gov/hmcts/reform/hmc/repository/LinkedGroupDetailsRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.LinkedGroupDetails;
+
+@Transactional(propagation = Propagation.REQUIRED)
+@Repository
+public interface LinkedGroupDetailsRepository extends CrudRepository<LinkedGroupDetails, Long> {
+
+    // TODO: implement DB query - https://tools.hmcts.net/jira/browse/HMAN-96
+    default void deleteHearingGroup(Long hearingGroupId) {
```

Comment:
is this a place holder for the next coming work?

Reply:
yes, this ticket is just to establish the endpoint and do the validation